go/libraries/doltcore/sqle/dprocedures/dolt_conflicts_resolve.go

```diff
@@ -362,9 +362,9 @@ func ResolveSchemaConflicts(ctx *sql.Context, ddb *doltdb.DoltDB, ws *doltdb.Wor
 
 	// TODO: schema names
 	tblSet := set.NewStrSet(tables)
```

Comment:
Probably could use a doltdb.TableNameSet here and get rid of the .Name use below? Not positive though